skt/kernelbuilder.py

```diff
@@ -218,6 +222,7 @@ def mktgz(self, timeout=60 * 60 * 12):
 
         with io.open(self.buildlog, 'wb') as writer, \
                 io.open(self.buildlog, 'rb') as reader:
+            self.__prepare_kernel_config(out=writer, err=subprocess.STDOUT)
```

Comment:
They are not exactly "standard" so I felt it would just add confusion, especially with other namespaces like `subprocess` and `sys`. But, if you feel it would make more sense I can change the names, it's not too much hassle

Reply:
I like `stdout` and `stderr` somewhat better. I think it makes it clearer what goes into those parameters. Please change, if you don't mind :)